assets/js/blocks/product-best-sellers/index.js

```diff
@@ -28,6 +29,20 @@ registerBlockType( 'woocommerce/product-best-sellers', {
 	attributes: {
 		...sharedAttributes,
 	},
+	transforms: {
+		from: [
+			{
+				type: 'block',
+				blocks: [
```

Comment:
Should we consider having a helper method that can return a list of blocks, minus the one being transformed to? If more blocks are added, each of these `transforms` will need to be updated.

Reply:
I wonder if we can do this automatically; get the list of blocks in the woocommerce category that have the rows attribute? I think that criteria will get the blocks we want to transform.

Alternately just having one single place where this is hardcoded would be easier to manage.

Reply:
Given that there is an order to calling `registerBlock` upon load, I don't think we can depend on dynamically querying block types. That list would start out empty for the first `woocommerce/...` block and would only be complete for the last.  I think we will need a static implementation for this.